xslt_src/common.xsl

```diff
@@ -145,7 +157,17 @@
 							<xsl:attribute name="font-weight">normal</xsl:attribute>
 							<xsl:attribute name="margin-bottom">12pt</xsl:attribute>
 						</xsl:if>
-						<xsl:text>Table </xsl:text>
+						
+						<xsl:if test="$namespace = 'iso' or $namespace = 'iec' or $namespace = 'itu' or $namespace = 'unece' or $namespace = 'unece-rec' or $namespace = 'nist' or $namespace = 'ogc' or $namespace = 'rsd' or $namespace = 'csa' or $namespace = 'csd'">
+							<xsl:text>Table </xsl:text>
+						</xsl:if>							
+						<xsl:if test="$namespace = 'gb'">
+							<xsl:choose>
+								<xsl:when test="$lang = 'zh'">表 </xsl:when>

```

Comment:
We should make this language-independent. Can you define these variables on top and we can override them from the XML later?